### DISCUSSION TITLE
Feature Request:  Add segmentation support for Yolov5

### DISCUSSION THREAD

**Discussion by kadirnar** [obss/sahi#771] [Ideas]
Hello @fcakyon ,

Added segmentation models to the Yolov5 library. Are you considering adding this support to the SAHI library?
https://github.com/kadirnar/Yolov7-SAHI/issues/3 @stphtan94117

**Comment by fcakyon** [comment]
We are currently working on a major refactorization for sahi==0.12. Most of the codebase will change after that. Then we can start working on yolov5 segmentation support :+1:

**Reply by ghost** [reply]
May I know if there is support yolov5 segmentation now?

**Reply by fcakyon** [reply]
There is no such support
